src/main/java/org/lwes/ArrayEvent.java

```diff
@@ -187,10 +189,10 @@ public void setEventName(String name) {
      */
     @Override
     public void set(String key, FieldType type, Object value) {
-        checkShortStringLength(key, encoding, MAX_FIELD_NAME_SIZE);
+        checkShortStringLength(key, MAX_FIELD_NAME_SIZE);
         if (ENCODING.equals(key)) {
             if (type == FieldType.INT16) {
-                setEncoding((Short) value);
+                setEncoding();
```

Comment:
If we're going to commit to always using UTF-8, we probably shouldn't bother calling setEncoding() again, even if someone is setting the "enc" field specifically.  Maybe we should throw an exception if anything other than UTF-8 is passed in here for "enc".  This code will currently allow someone to think they've successfully set the encoding to ISO_8859_1 by setting the "enc" field directly, but the serialized event will be UTF-8

Reply:
I think we should settle on UTF-8 and remove the encoding structures all together.